We have more:

> Expected command to pass but it did not.\r\nFile Name: F:\workspace\_work\1\s\.dotnet\dotnet.exe\r\nArguments: msbuild /t:Build F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj /restore /p:RuntimeIdentifier=win10-x64 /p:TestRuntimeIdentifier=win10-x64\r\nExit Code: 1\r\nStdOut:\r\nMicrosoft (R) Build Engine version 16.5.0+d4cbfca49 for .NET Core\r\nCopyright (C) Microsoft Corporation. All rights reserved.\r\n Restore completed in 52.04 ms for F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\LibraryWithRids\LibraryWithRids.csproj.\r\n Restore completed in 51.95 ms for F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\LibraryWithoutRid\LibraryWithoutRid.csproj.\r\n Restore completed in 51.96 ms for F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj.\r\n Restore completed in 51.95 ms for F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\LibraryWithRid\LibraryWithRid.csproj.\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: The "ResolvePackageAssets" task failed unexpectedly. [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: System.IO.FileNotFoundException: Could not load file or assembly 'NuGet.ProjectModel, Version=5.6.0.5, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: File name: 'NuGet.ProjectModel, Version=5.6.0.5, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.Tasks.ResolvePackageAssets.CacheReader.CreateReaderFromMemory(ResolvePackageAssets task, Byte[] settingsHash) [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.Tasks.ResolvePackageAssets.CacheReader..ctor(ResolvePackageAssets task) in //src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs:line 477 [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.Tasks.ResolvePackageAssets.ReadItemGroups() in //src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs:line 301 [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.Tasks.ResolvePackageAssets.ExecuteCore() in //src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs:line 294 [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.T

--- a/global.json
+++ b/global.json
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.1.202",
+    "dotnet": "3.1.300-servicing-015161",
     "vs-opt": {
       "version": "15.9"
     }

--- a/global.json
+++ b/global.json
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.1.300-servicing-015161",
+    "dotnet": "3.1.301-servicing-015165",
     "vs-opt": {
       "version": "15.9"
     }

